src/Behat/Model/OperationRequest.php

```diff
@@ -107,7 +107,29 @@ public function getFormatted(): array
 
     private function formatFilters(): string
     {
-        return '';
+        $filters = $this->getFilters();
+
+        if (count($filters) <= 0) {
+            return '';
+        }
+
+        $filtersData = '';
+
+        /**
+         * @var string $filterName
+         * @var mixed $filterValue
+         */
+        foreach ($filters as $filterName => $filterValue) {
+            $filtersData .= sprintf(
+                "%s: %s,\n",
```

Comment:
Should we add `,\n` also to the last one?

Reply:
GraphQL query allows leaving trailing comma